internal/client/networking.go

```diff
@@ -37,6 +41,45 @@ func ReserveIPCount(count int64) IPReservationTypeFunc {
 	}
 }
 
+// ReserveIPRange configures the IP reservation to reserve a range of IP addresses.
+// A range out of two IPs separated by a hyphen.
+func ReserveIPRange(ipRange string) (IPReservationTypeFunc, error) {
```

Comment:
It may be confusing reading this function name when there is already `ReserveIP`, maybe `GetIPReservationTypeFunc`?

Reply:
Done.